src/stmt.rs

```diff
@@ -36,8 +36,10 @@ pub enum Statement {
     #[serde(rename = "quota")]
```

Comment:
If there are more entries that are going to be added in the future, should this enum maybe be marked as `non_exhaustive`?

Reply:
Good point. Addressed in #70 and I think that `non-exhaustive` should be added in v0.5.0.